### DISCUSSION TITLE
how can i separate filter by function from header text or v-text-field in Vuetify v-data-table?

### DISCUSSION THREAD

**Discussion by ShahanurSharif** [vuetifyjs/vuetify#15654] [Q&A]
As you can see in this codepen link, I have replaced the header text with v-text-field but the problem is the v-text-field is attached with filterBy icon (the up-down arrow icon) as well. When i click on the text-field the filterBy function is working that i dont want. So far I think the @click event is covering whole area. I wish to separate the text-field and filterBy behavior. Any help will be greatly appreciated.

https://codepen.io/MicroDreamIT/pen/vYRvMza

**Comment by KaelWD** [comment]
https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation

**Reply by ShahanurSharif** [reply]
@KaelWD many many thanks for your reply and appreciate it, I wish to have a more detailed answer to fix the bug.
